### PR TITLE
Eeyun/release v70

### PR DESCRIPTION
**Pull request by eeyun**

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,26 +22,42 @@
 - A sad goodbye [#5873](https://github.com/habitat-sh/habitat/pull/5873) ([elliott-davis](https://github.com/elliott-davis))
 - Update version [#5880](https://github.com/habitat-sh/habitat/pull/5880) ([smacfarlane](https://github.com/smacfarlane))
 - Add 0.69.0 release post [#5877](https://github.com/habitat-sh/habitat/pull/5877) ([smacfarlane](https://github.com/smacfarlane))
+
+#### New Features & Enhancements
```
Comment (christophermaier) [inline CHANGELOG.md]: New features should be the first section, since it's the highest impact.

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,26 +22,42 @@
...
+- Add simple auth and TLS to supervisor HTTP gateway [#5874](https://github.com/habitat-sh/habitat/pull/5874) ([raskchanky](https://github.com/raskchanky))
+
+## [0.69.0](https://github.com/habitat-sh/habitat/tree/0.69.0) (2018-11-27)
```
Comment (christophermaier) [inline CHANGELOG.md]: The issues here don't line up with https://github.com/habitat-sh/habitat/blob/0.69.0/CHANGELOG.md, which is what I'd expect to see. Some 0.70.0 issues are down here.

Comment (eeyun) [reply CHANGELOG.md]: Hmm strange, i got that list from the `git log --pretty=oneline --merges 0.68.0..0.69.0` I'll correct these momentarily.

**Pull request merged**